app/breaksPlanner.js

```diff
@@ -168,34 +168,42 @@ class BreaksPlanner extends EventEmitter {
     this.emit('updateToolTip')
   }
 
-  skipToMicrobreak () {
+  scheduleMicrobreak (delay) {
     this.scheduler.cancel()
     const shouldBreak = this.settings.get('break')
     const shouldMicrobreak = this.settings.get('microbreak')
-    const breakInterval = this.settings.get('breakInterval') + 1
     if (shouldBreak && shouldMicrobreak) {
+      const breakInterval = this.settings.get('breakInterval') + 1
```

Comment:
any reason why this line was moved?

Reply:
1. For consistency with `skipToBreak` (I compared the functions to check for any variations in behaviour).
2. Because the call to `this.settings.get` is unnecessary unless `shouldBreak && shouldMicrobreak` is true.